tsconfig.json

```diff
@@ -13,7 +13,7 @@
         "noImplicitAny": false,
         "moduleResolution": "node",
         "lib": [
-            "es5",
+            "es2015",
```

Comment:
Had to change this so I could use `Symbol.toStringMap` and `Symbol.iterator` in the typings.